core/extensions.js

```diff
@@ -114,7 +114,7 @@ Blockly.Extensions.registerMutator = function(name, mixinObj, opt_helperFn,
       if (!Blockly.Mutator) {
         throw Error(errorPrefix + 'Missing require for Blockly.Mutator');
       }
-      this.setMutator(new Blockly.Mutator(opt_blockList));
+      this.setMutator(new Blockly.Mutator(opt_blockList || []));
```

Comment:
Functional change, Mutator does not accept null or undefined

Reply:
What did it do before?

Reply:
Well it didn't handle not passing a block list. It would have thrown [here](https://github.com/google/blockly/blob/master/core/mutator.js#L134). However, a call to ``checkMutatorDialog_`` guards this code, and that checks to see if the compose / decompose methods exist, if you register a mutator with a compose / decompose and don't pass a block list it will crash.

Tried it and indeed it does.